src/main/java/net/sf/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -49,24 +49,24 @@
     private final FieldContentParser parser;
 
 
-    public LatexFieldFormatter() {
-        this(true);
+    public LatexFieldFormatter(JabRefPreferences prefs) {
+        this(true, prefs);
     }
 
-    private LatexFieldFormatter(boolean neverFailOnHashes) {
+    private LatexFieldFormatter(boolean neverFailOnHashes, JabRefPreferences prefs) {
```

Comment:
Conceptually, yes. There are two problems (in general): 

1. that configuration object is in many cases a running target. You pass something that may or may not be used five calls from here. Layout is a good example, where some formatters use some preferences and you have to inject that in lots of methods. Adding a new formatter may change the object and so on.

2. That configuration object will depend on the preferences, so I do not really see where this disconnection should happen. At least not if the configuration of object should configure itself based on the preferences.